task(connector-sdk): Adding Apache Hive Example (#152)
fivetran/fivetran_connector_sdk: README.md

<summary><h3>Source Examples</h3></summary>

- [apache_hbase](https://github.com/fivetran/fivetran_connector_sdk/tree/main/examples/source_examples/apache_hbase) - This is an example of how we can connect and sync data from Apache HBase by using Connector SDK. It uses happybase and thrift libraries to connect to HBase and fetch data.
- [apache_hive/using_pyhive](https://github.com/fivetran/fivetran_connector_sdk/tree/main/examples/source_examples/apache_hive/using_pyhive) - This is an example of how we can sync data from Apache Hive by using Connector SDK using PyHive.

> - [apache_hive/using_sqlalchemy](https://github.com/fivetran/fivetran_connector_sdk/tree/main/examples/source_examples/apache_hive/using_sqlalchemy) - This is an example of how we can sync data from Apache Hive by using Connector SDK using SQLAlchemy with PyHive.

Reviewer: ```suggestion
- [apache_hive/using_sqlalchemy](https://github.com/fivetran/fivetran_connector_sdk/tree/main/examples/source_examples/apache_hive/using_sqlalchemy) - This example shows how you can sync data from Apache Hive by using Connector SDK and SQLAlchemy with PyHive.
```
